mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/ThreadMirror.cs

```diff
@@ -168,6 +168,8 @@ public static void FetchFrames(IList<ThreadMirror> threads)
 				throw new ArgumentNullException ("loc");
 			try {
 				vm.conn.Thread_SetIP (id, loc.Method.Id, loc.ILOffset);
+				InvalidateFrames ();
```

Comment:
Same here, that doesn't look like a no-op, we probably want to do this only if the server supports version `2.52`.